src/python/merge_tester.py

```diff
@@ -70,7 +70,7 @@
 )
 
 
-def write_cache(
+def write_cache_merge_status(
     status: MERGE_STATES, runtime: float, explanation: str, cache_file: str
```

Comment:
Nit: 

Here are three variants of the word "run time" in computer science:
 * run-time -- adjective ("the run-time performance")
 * run time -- noun -- a moment in time ("occurs at run time"), or an amount of time ("the run time is 8 hours")
 * runtime -- noun -- a program that runs/supports another program ("the runtime handles memory allocation"); a synonym for "runtime system"

So I would make this run_time because it corresponds to the English "run time".  But this isn't a huge deal either way.